tox.ini

```diff
@@ -20,7 +20,7 @@ setenv = VIRTUAL_ENV={envdir}
 deps =
     -r{toxinidir}/requirements.txt
     -r{toxinidir}/test-requirements.txt
-commands = stestr run --serial --test-path {[testenv]unit_tests} {posargs}
```

Comment:
Guess you could have dropped the --serial too but same result

Reply:
I figured that we could drop that dependency. I don't like having too many dependencies, the more you have the more things can break 😉